Manual.md

```diff
@@ -447,10 +449,9 @@ Multiple licenses should be separated by commas, Example: `GPL-3.0-or-later, cus
   Note: `MIT`, `BSD`, `ISC` and custom licenses
   require the license file to be supplied with the binary package.
 
-- `maintainer` A string in the form of `name <user@domain>`.  The
-  email for this field must be a valid email that you can be reached
-  at.  Packages using `users.noreply.github.com` emails will not be
-  accepted.
+- `maintainer` A string in the form of `name <user@domain>`.  The email for this field
```

Comment:
Any content change here?

Reply:
No, just fixing indentation to match the other items in the block.